apps/console/src/public/deployment.config.json

```diff
@@ -341,6 +341,7 @@
                 }
             },
             "applications": {
+                "saasMyAccount": false,
```

Comment:
Let's add this as a disabled feature to `disabledFeatures` array. 

Also let's send a PR to framework as well.

Reply:
Get a pull as well. I recently added a feature flag for SaaS My Account settings page as well.

https://github.com/wso2/identity-apps/pull/6094/files#diff-e66c115a503827bbd4e1bd6ad04d6ff16ed77f158ae3608fae0f786905382f63R347

Reply:
Added a config to disabled features list `myAccountSimplifiedSettings`